### PR TITLE
[BREAKING] Multi region support

### PR DESCRIPTION
**Pull request by CalleB3**
Closing this PR, 

We decided to start from scratch, some too quick decisons were made in this PR.

### CODE REVIEW

File: modules/azure/aks-global/aad-group.tf [hunk 1]
```diff
@@ -1,35 +1,35 @@
 resource "azuread_group" "view" {
   for_each     = { for ns in var.namespaces : ns.name => ns }
-  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${each.value.name}${var.group_name_separator}view"
+  display_name = "${var.aks_group_name_prefix}-${var.subscription_name}-${var.environment}-${each.value.name}-view"
```
Comment (simongottschlag) [inline modules/azure/aks-global/aad-group.tf:3]: @CalleB3 why are we removing the variable for the separator?

**Review by simongottschlag — CHANGES_REQUESTED** [review]

File: modules/azure/aks-global/aad-group.tf (hunk 1 above)
Comment (jimgus) [reply modules/azure/aks-global/aad-group.tf:3]: I think it is a good idea to remove it unless we really need to be able to change the separator. Improves readability a lot.

Comment (simongottschlag) [reply modules/azure/aks-global/aad-group.tf:3]: @jimgus quite sure it's being used.

Comment (jimgus) [reply modules/azure/aks-global/aad-group.tf:3]: @simongottschlag  Oh, that is a pity in that case

**Pull request closed without merging**